internal/scti/chain_validation_test.go

```diff
@@ -690,7 +692,7 @@ func TestPreIssuedCert(t *testing.T) {
 				t.Fatalf("failed to ValidateChain: %v", err)
 			}
 			for i, c := range chain {
-				t.Logf("chain[%d] = \n%s", i, x509util.CertificateToString(c))
+				t.Logf("chain[%d] = \n%s", i, md5.Sum(c.Raw))
```

Comment:
md5?

Would printing the `Subject` make it easier to tie back to the certs in the chain?